test/compflags/diten/unusedFunction.good

```diff
@@ -0,0 +1 @@
+unusedFunction.chpl:3: baz
```

Comment:
I think it would be useful to start the list of unused functions with a label or header, in case other output in generated.  It might also be good to label the end of the list.  Something like
```
These are the unused functions in your program:
unusedFunction.chpl:3: baz
There are no more unused function
```
or
```
These are the unused functions in your program:
unusedFunction.chpl:3: baz
End of unused functions list
```

Reply:
What do you think about indenting the list instead of having an ending message?

```
The following functions are unused:
  unusedFunction.chpl:3: baz
```

Reply:
That works for me!